src/els_methods.erl

```diff
@@ -151,16 +151,12 @@ initialize(Params, State) ->
               els_formatting_provider:is_enabled_document()
           , documentRangeFormattingProvider =>
               els_formatting_provider:is_enabled_range()
-          , documentOnTypeFormattingProvider =>
-              els_formatting_provider:is_enabled_on_type()
+          %%, documentOnTypeFormattingProvider =>
+          %%    els_formatting_provider:is_enabled_on_type()
```

Comment:
Given the bug that is being fixed, this one should be ok, it just signals that it is not supported. It is in fact a valid capability.

Reply:
If I understood this correctly, IntelliJ implements version `3.14` of the protocol (understandable, since `3.15` just came out) and in that version this field cannot be a `boolean`, but it has to be a _struct_. This is addressed in `3.15`. I guess it all depends if we want to consider that as a protocol bug and/or support IntelliJ.
@alanz Is this the bug you are referring to, or is it something else?

Reply:
I saw the bug report as invalid capabilities returned. I know the one is invalid (with my longstanding comment), was not sure of the other. If it is a problem it may as well stay commented out, it has the same practical interpretation.